### ISSUE TITLE
drupal_test_running() missing in drupal-7.50.1.tar.gz

### ISSUE THREAD

**Issue by petrowsky** [omega8cc/boa#1018]
When building my custom site, the last build I tried from files.aegir.cc was 7.43.1.

When working with 7.50.1 I'm getting a fatal error. 

```Error: Call to undefined function drupal_test_running() in /data/disk/o1/static/custom-site/includes/common.inc, line 5726 [3.75 sec, 12.91 MB]```

When heading into the directory and doing the following, I only get two lines for **drupal_test_running**.

```
o1@s1:~/static/custom-site-7.50$ grep -r 'drupal_test_running' *
includes/common.inc:  if (drupal_test_running()) {
modules/simpletest/drupal_web_test_case.php:    drupal_test_running(TRUE);
```

Compare that to...

```
o1@s1:~/static/custom-site-7.43$ grep -r 'drupal_test_running' *
includes/common.inc:  if (drupal_test_running()) {
includes/bootstrap.inc:function drupal_test_running($set = NULL) {
modules/simpletest/drupal_web_test_case.php:    drupal_test_running(TRUE);
```

Looks like some patch may have failed being applied? There is no **function drupal_test_running()** in the 7.50 files.

**Comment by omega8cc** [comment]
I think it was not tested yet, and some conflicts were resolved without proper cleanup. Good catch.. thanks!

**Comment by omega8cc** [comment]
This should be fixed now (hotfix applied to 7.50.1)
